webpack/stories/docs/adding-new-components.stories.mdx

```diff
@@ -114,23 +114,51 @@ react_component(component_name, props)
 **Example:**
 
 ```erb
-<%= react_component('PowerStatus', id: host.id, url: power_host_path(host.id)) %>
+<div id="my-cool-power-status">
+  <%= react_component('PowerStatus', id: host.id, url: power_host_path(host.id), errorText: 'N/A') %>
+</div>
 ```
 
 will render the following HTML:
 
 ```html
-<foreman-react-component
-  name="PowerStatus"
-  data-props="<%= {
-    id: host.id,
-    url: power_host_path(host.id)
-  }.to_json %>"
-></foreman-react-component>
+<div id="my-cool-power-status">
+  <foreman-react-component
+    name="PowerStatus"
+    data-props="<%= {
+      id: host.id,
+      url: power_host_path(host.id),
+      errorText: 'N/A',
+    }.to_json %>"
+  ></foreman-react-component>
+</div>
 ```
 
 (Note that the React component is rendered as a [web component](https://developer.mozilla.org/en-US/docs/Web/Web_Components).)
 
+### Changing the props from legacy JS
+
+We allow changing the root component props from the legacy JS.
+Be aware, that this will re-render the component, so should be used with care!
```

Comment:
This feature should only be used for limited use cases:

1. A legacy JS library/component needs to talk to a React component, AND
2. The component is simple enough that it wouldn't otherwise make sense to store its data in the Redux store.

I think it would be good to call that out here, and say it's not recommended to use this otherwise.